sources/us/ca/san_benito.json

```diff
@@ -0,0 +1,23 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "06069",
+            "name": "San Benito County",
+            "state": "California"
+        },
+        "country": "us",
+        "state": "ca",
+        "county": "San Benito"
+    },
+    "data": "https://s3.amazonaws.com/data.openaddresses.io/cache/uploads/nvkelso/49c92f/SBCAddress.zip",
+    "type": "http",
+    "compression": "zip",
+    "conform": {
+        "type": "shapefile",
+        "id": "APN",
+        "number": "ADDRESS",
+        "street": "STNA_FULL",
+        "postcode": "ZIPCODE",
+        "unit": "UNIT"
```

Comment:
The `CITY` field uses three letter codes like `HOL` and `SJB` instead of Hollister or San Juan Bautista. Should those be included?